backendParent/docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3'
```

Comment:
why is it version 3?

Reply:
https://docs.docker.com/compose/compose-file/
kindly refer this page